crates/core/src/types/payload.rs

```diff
@@ -0,0 +1,158 @@
","0xf86e57843baa0c40825208944dc4ec6ac43c8c45777292db987203c0248e17b7880429d069189e0000808360306ca0c0400263ff52d68fde756b45706675f6e49a0607168babae113cdd12bc8cb7cfa0547239a432debca6b22a10809c5b41540b3c0d64f50c60575bc66a9d68f012cf","0xf86e58843baa0c40825208940d2f39f251cb547cba567a31e5e9f93c19dffa85880429d069189e0000808360306ca0c9395d097cb0d65bba0fb2e75b199dac4b78c10f154a09cfe85848c65cd842afa04b4214494817a7b8b8801cceec9d53245035d12dbf17d1c0b2d09f77dad3bed1","0xf86e59843baa0c40825208949eb31fb94ce5111e2a04cb9d156b513887ccbd00880429d069189e0000808360306ba03c01e7b5fea33c8a1fefa89718565b0bd45eee09e304ca58343fded450aef27ea06138a496c273164a4761c1a9f2bbc71a82e9ac46882c3450a547e19f70e67ab8","0xf86e5a843baa0c408252089404b88ef83f8c41b1465d360a1e82f07ae190892a880429d069189e0000808360306ba0056b844b2d3fba16f16433f818f7568b185a51d04416bab8ae1637c02023fbcaa0205ca5d43fe235bd6a5a6b0a6e5e2e9816454a4a641679c4799d460cebec86c5","0xf86e5b843baa0c4082520894af23e04b04fbe15630eadd32a6f27a5a65ea554a880429d069189e0000808360306ca031f2c170416153f6a5ec0861dd60285af1051474536d6d94e15016a432098914a07031c684a3e8f81497942050cddb2b13ea7965ac3b47ca416500b0df3ec965a0","0xf86e5c843baa0c4082520894746cdff371e3f1e905b3ac52280078bac2dec7dd880429d069189e0000808360306ca0e9b1f8d21e043401d00a19702a54f58a6f9afd431f229465f2ae6f221c393988a0531b5cc672f4e70603e3d5b3e57ccd21863f36531a5f4e9eb90e78238bac2702","0xf86e5d843baa0c4082520894c33e5155bdbf1a0a7ceb1b80f8586c5cda5c3781880429d069189e0000808360306ca0e2c3e9f304c52cc916686cd7b4c85bd8ac2f63256ab937d4d08d9314bebd9f68a07b69d30bb58ca0f8dd832193cb4e0484e5f78ab6295877d8e75025e4ad37b3df","0xf86e5e843baa0c4082520894e7fdef5f5219068f3d0f88a7445005574c662798880429d069189e0000808360306ca0262f19b3634b81c804ee5aea089c56d05122ab399818c510f706f5bef52f8162a05cd48af19c4b214a08c35041c7e4f80b479f46e8099a42702961e9250e7df8f0","0xf86e5f843baa0c4082520894f0a81a63c5e09b0bd08e027de48058e377d3732d880429d069189e0000808360306ba02ea289034396bf771573e2dd0982a45fc13d948b29883ca4ab7177137e03284da015ac417bce04f3aff06640b0ba341f42fc0f7f7ef18ee641161eb7293dc8fdaa","0xf86e60843baa0c40825208949878ab34dc3b4a63c80fdb733491472c11d59a56880429d069189e0000808360306ca055e6d423d924fce58749f9ea3499f2ad8f8cb39897108c85ec38e43637188dc7a05f9bf9d64ecfc9d4a05f7371343974098741fbcb22f90176671e56804420b916","0xf86e61843baa0c4082520894912859bebae3086ac7a062dee5d68aa8ed2d71ec880429d069189e0000808360306ca03edf60bc0d47f457bb68fb8b2ad6208a965452daf526dfd90fcf81680f8270bda072a54309f9e3a31128471778083d0f25097e665a8727a1dc1a8fa9084be55538","0xf86e62843baa0c40825208945a0b737ed85049410e5ea61f444d07d5c8c0359f880429d069189e0000808360306ba05018a457401820eee3b51b9a4d68f88c77c11adde86292b807185237717bc3c5a05b7f3a142b8489fdeb32672ef031cffb0f19665b934abc029b4f9ce99574e7c5","0xf86e63843baa0c4082520894305a5dfd46e6128abce28c03b3ad971f4e4915ff880429d069189e0000808360306ba0d0967bf689588b457266342fce9ed9c7a0e6954d9f56f861b48b0f1f1f2f9875a0687b8ac671cb3036e2283113490d4e14cab64f926968c4387b7a392983a11923"],"withdrawals":[]}"#;
```

Comment:
maybe a smaller example 😅 ?

Reply:
Yes, it is indeed quite large, I can remove some transactions from it to make it smaller